packages/devtools/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
-## 0.1.5 - TBD
+## 0.1.5 - 2019-08-05
 * Support expanding or collapsing all values in the Call Tree and Bottom Up views (parts of the CPU profiler).
 * Support touchscreen scrolling and selection in flame charts.
+* Display structured error messages in the Logging view.
```

Comment:
Nit: make it clear this is still off by default.
Perhaps add:
when "show structured errors" is enabled.

Reply:
Done.